Bottom: 4c44eaa5aaa7557e873046cfa7c812e97c9e6217
Top:    a3ca278d03ae3f1e19083582cffaadc7e2c2dfab
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 15:39:12 -0300

Refresh of mallopt-detailed-probes.patch

---

diff --git a/malloc/malloc.c b/malloc/malloc.c
index 4c40c2f..45e5208 100644
--- a/malloc/malloc.c
+++ b/malloc/malloc.c
@@ -4685,7 +4685,7 @@ int __libc_mallopt(int param_number, int value)
     if (value >= 0 && value <= MAX_FAST_SIZE)
       {
 	LIBC_PROBE (memory_mallopt_mxfast, 2, value, get_max_fast ());
-	set_max_fast(value);
+	set_max_fast (value);
       }
     else
       res = 0;
